docs/source/dev/index.rst

```diff
@@ -103,10 +103,18 @@ generally debugging and building out features.
 Other Documentation
 ~~~~~~~~~~~~~~~~~~~
 Additional documentation exists that may be useful for developers seeking deeper
-understanding of the solver and mathematics.  This documentation is not generally
-necessary for most development efforts.
-
-.. toctree::
-   :maxdepth: 1
-
-   other_docs.rst
+understanding of the solver and mathematics.
+
+- `NWTC Programmer’s Handbook <https://drive.google.com/file/d/1bDV1fBkiZUWs6Tkzb6nhCMUQvHpN_OtM/view?usp=sharing>`_
+   This is an overview of programming guidelines for FAST 8. While some syntax and minor details have
+   changed in OpenFAST, most of this guide is still relevant.
+- :download:`DCM_Interpolation.pdf <../../OtherSupporting/DCM_Interpolation/DCM_Interpolation.pdf>`
+   This is a summary of the mathematics used in the interpolation of 
+   DCM (direction cosine matrices) using logarithmic mapping and matrix exponentials.
+- :download:`OpenFAST_Algorithms.pdf <../../OtherSupporting/OpenFAST_Algorithms/OpenFAST_Algorithms.pdf>`
+   This is a summary of the solve method used in the glue code.
+- :download:`OutListParameters.xlsx <../../OtherSupporting/OutListParameters.xlsx>`
```

Comment:
This document is useful for both developers and users.  I'm not sure where it should go, but perhaps we could link to it in section 4 of readthedocs as well.